airbyte-ci/connectors/auto_merge/src/auto_merge/env.py

```diff
@@ -0,0 +1,8 @@
+# Copyright (c) 2024 Airbyte, Inc., all rights reserved.
+
+from __future__ import annotations
+
+import os
+
+GITHUB_TOKEN = os.environ["GITHUB_TOKEN"]
+PRODUCTION = os.environ.get("AUTO_MERGE_PRODUCTION", "false").lower() == "true"
```

Comment:
Not required for this PR, but we have various ways that we define truthy in airbyte-ci. At some point we should probably try to make them consistent.